pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -24,8 +24,11 @@ import (
 )
 
 const (
-	rootDir             = "/opt/tectonic"
-	defaultReleaseImage = "registry.svc.ci.openshift.org/openshift/origin-release:v4.0"
+	rootDir = "/opt/tectonic"
+
+	// TODO: This should be decided by installer, not hard-coded
```

Comment:
@smarterclayton how are we supposed to handle this? :-) Also what is the preferred registry for our images? The RHCOS seems to have RH registry hardcoded as default, so using the `:latest` tag does not work and we really don't want to pin this into stable tag, otherwise we won't be able to test the changes in our operator images....

I guess we want all images in quay.io at some point but we want a knob to switch that to registry.svc.ci.openshift.org for the CI?